File: Discord Bot Codes/python/Creating a Button.py
import discord
from discord.ext import commands

bot = commands.Bot(command_prefix='!')

@bot.event
async def on_ready():
    print(f'Logged in as {bot.user.name}')

@bot.command()
async def button(ctx):
    button = discord.ui.Button(style=discord.ButtonStyle.primary, label="Click Me!")

    await ctx.send("Click the button below:", view=discord.ui.View().add_item(button))

@bot.event
async def on_button_click(interaction):
    if interaction.custom_id == "button":
        await interaction.response.send_message("Button clicked!")

bot.run('YOUR_BOT_TOKEN')

File: Discord Bot Codes/Javascript/Creating a Button.js
const { Client, Intents, MessageActionRow, MessageButton } = require('discord.js');

const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });

client.on('ready', () => {
  console.log(`Logged in as ${client.user.tag}`);
});

client.on('messageCreate', async (message) => {
  if (message.content === '!button') {
    const row = new MessageActionRow().addComponents(
      new MessageButton()
        .setCustomId('button')
        .setLabel('Click Me!')
        .setStyle('PRIMARY')
    );

    await message.channel.send({ content: 'Click the button below:', components: [row] });
  }
});

client.on('interactionCreate', async (interaction) => {
  if (!interaction.isButton()) return;

  if (interaction.customId === 'button') {
    await interaction.reply('Button clicked!');
  }
});

client.login('YOUR_BOT_TOKEN');
